### PR TITLE
[Bahasa Indonesia] - ID Translation of fundamentals/getting-started/

### PR DESCRIPTION
**Pull request by abdshomad**
Hi @petele, 
CC: @tyohan, @mychaelgo, 

I have finished my next Indonesian Translation of Web Fundamentals for fundamentals/getting-started/ section. They consists of 16 files, where 14/16 files are include files. 

I have tested it running on my local development version. Here's the screenshot:
getting-started

Please review it.

### CODE REVIEW

File: src/content/id/_book.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_book.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_common-links.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% comment %}
```
Comment (petele) [inline src/content/id/_common-links.md]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_project.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_project.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_redirects.yaml [hunk 4]
```diff
@@ -0,0 +1,338 @@
+redirects:
```
Comment (petele) [inline src/content/id/_redirects.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Fundamentals.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Fundamentals.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Showcase.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Showcase.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Tools.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Tools.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/_upper_tabs-Updates.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/_upper_tabs-Updates.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/_book.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/fundamentals/_book.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/_redirects.yaml [hunk 10]
```diff
@@ -0,0 +1,137 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/fundamentals/_redirects.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/getting-started/_redirects.yaml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
```
Comment (petele) [inline src/content/id/fundamentals/getting-started/_redirects.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/fundamentals/getting-started/_toc.yaml [hunk 12]
```diff
@@ -0,0 +1,37 @@
+toc:
```
Comment (petele) [inline src/content/id/fundamentals/getting-started/_toc.yaml]: Please remove this file - it shouldn't be included in the localized folders.

File: src/content/id/manifest.json [hunk 13]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (petele) [inline src/content/id/manifest.json]: Please remove this file - it shouldn't be included in the localized folders.

**Comment by abdshomad** [comment]
@petele - Included files are deleted. Please re-check.

**Pull request merged**